lib/modules/manager/cocoapods/extract.ts

```diff
@@ -141,6 +127,15 @@ export async function extractPackageFile(
           groupName,
           skipReason: 'path-dependency',
         };
+      } else {
+        dep = {
+          depName,
+          groupName,
+          datasource: PodDatasource.id,
+          currentValue: currentValue ?? '>= 0',
```

Comment:
Does this _need_ to be `>= 0`? I think we should find a way to avoid needing this type of massage